src/ngraph/runtime/cpu/cpu_external_function.cpp

```diff
@@ -1433,6 +1476,12 @@ void runtime::cpu::CPU_ExternalFunction::build(ngraph::pass::PassConfig& pass_co
         op_names.push_back(node->get_name());
         handler->second(this, node.get(), in, out);
 
+        if (do_ngraph_trace != nullptr)
+        {
+            tv_inputs.push_back(in);
+            tv_outputs.push_back(out);
```

Comment:
Please extend OpAttributes to capture shape/size of these tensors in m_op_attrs.

Reply:
Oh, I wanted to do this but was a little afraid to change the struct.